docs/roleassumption.md

```diff
@@ -1,21 +1,21 @@
 # Creating clusters using cross account role assumption using KIAM
```

Comment:
Should we add a link at the top of this file to the same doc in the `release-0.3` branch?

Reply:
That wouldn't hurt, are we serving docs from release-0.3 branch in this repo?

Reply:
We don't publish the docs anywhere other than from the git repo itself. Until we do, we should likely ensure that we add links to the `release-0.3` doc to avoid user confusion when trying to consume releases.

Reply:
How would the link look like? Is that preferred to just revert the files to what they were before?

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/release-0.3/docs/roleassumption.md for this specific doc

Reply:
I'd like to avoid deferring docs changes, otherwise we risk missing more things than if we update in sync with the changes.

Reply:
Added, ptal :)